.codeclimate.yml

```
@@ -9,6 +9,9 @@ plugins:
   rubocop:
     enabled: true
     channel: rubocop-1-56-3
+    checks:
+      Rails/ActionControllerFlashBeforeRender:
```

Looks like this is the correct version:
```suggestion
      Rubocop/Rails/ActionControllerFlashBeforeRender:
```